examples/static_map/custom_type_example.cu

```diff
@@ -25,32 +25,13 @@
 #include <cuda/functional>
 
 // User-defined key type
-#if !defined(CUCO_HAS_INDEPENDENT_THREADS)
```

Comment:
Are we removing this switch to make the example less complex?

Reply:
The new map doesn't support key types larger than 8 bytes.